src/main/java/ai/zuva/docai/classification/ClassificationRequest.java

```diff
@@ -111,6 +116,23 @@ public ClassificationResult getStatus() throws DocAIClientException, DocAIApiExc
         "api/v2/classification/" + requestId, 200, ClassificationResult.class);
   }
 
+  /**
+   * Get multiple classification statuses and results
+   *
+   * @return A ClassificationMultipleResults object, containing the statuses of all requests and, if
+   *     available, the results
+   * @throws DocAIClientException Unsuccessful response code from server
+   * @throws DocAIApiException Error preparing, sending or processing the request/response
+   */
+  public ClassificationMultipleResults getStatuses()
```

Comment:
This should be `static`, like `createRequest`, with two versions taking either `ClassificationRequest[] classificationRequests` or `string[] classificationRequestIds` as an argument. It's not really a method on one instantiated `ClassificationRequest` object, it's just tucked into this class for code organization purposes.

Reply:
bbd36c32834795976fee5f2313b9e0830be9232c